src/main/java/io/split/android/engine/matchers/BetweenMatcher.java

```diff
@@ -51,12 +51,10 @@ public boolean match(Object matchValue, String bucketingKey, Map<String, Object>
 
     @Override
     public String toString() {
-        StringBuilder bldr = new StringBuilder();
-        bldr.append("between ");
-        bldr.append(_start);
-        bldr.append(" and ");
-        bldr.append(_end);
-        return bldr.toString();
+        return "between " +
```

Comment:
I am seeing in several files this way of concatenating string instead of having one line. Can we refactor this to be in one line or is something that your linter is requesting?

Reply:
Hey @mmelograno, thank you for the review. I've updated what you told me.